webserver/script/CostBenefitAnalysis/preprocessing_loadprofiles/split_file.py

```diff
@@ -86,21 +89,22 @@ def split_file(county, scenarios={'Scenario 1': 'BaseCase'}, controlled_types=['
         for Management in controlled_types:
             for county in county:
                 for SLAC_Year in years:
-                    # Read SLAC Driver Count
-                    Driver_count_file_name = "{}_{}_weekend__driver_counts.csv".format(Scenario, SLAC_Year)
-                    driver_count_df = pd.read_csv(os.path.join(driver_counts_path, Driver_count_file_name))
 
-                    if Scenario in ['WorkPublic', 'FastPublic', "Work", 'Equity']:
+                    # Read SLAC Driver Count
+                    Driver_counts_file_name = "{}_{}_weekday__driver_counts.csv".format(Scenario, SLAC_Year)
+                    driver_count_df = pd.read_csv("{}/{}/{}".format(bucket, driver_counts_path, Driver_counts_file_name))
 
+                    if Scenario in ['WorkPublic', 'FastPublic', 'Work', 'Equity']:
                         weekday_file_name = "{}_rescaled_{}_weekday_{}_county_{}_load.csv".format(Scenario, SLAC_Year, county, Management)
                         # weekend_file_name = "{}_rescaled_{}_weekend_{}_county_{}_load.csv".format(Scenario, SLAC_Year, county, Management)
                     else:
                         weekday_file_name = "{}_{}_weekday_{}_county_{}_load.csv".format(Scenario, SLAC_Year, county, Management)
                         # weekend_file_name = "{}_{}_weekend_{}_county_{}_load.csv".format(Scenario, SLAC_Year, county, Management)

```

Comment:
can we remove this comment and the one above?

Reply:
removed